### PR TITLE
Add Devise recoverable columns to users table

### PR DESCRIPTION
**Pull request by stevenwilkin**
Migration pulled out of [output#143](https://github.com/guidance-guarantee-programme/output/pull/143)

### CODE REVIEW

File: db/migrate/20150427100026_add_devise_recoverable_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddDeviseRecoverableToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :reset_password_token, :string
```
Comment (andrewgarner) [inline db/migrate/20150427100026_add_devise_recoverable_to_users.rb]: Should there be an index? I notice there is one for the confirmation token in the schema.

Comment (stevenwilkin) [inline db/migrate/20150427100026_add_devise_recoverable_to_users.rb]: Good spot!

**Comment by stevenwilkin** [comment]
Updated to include index. Laziness will be the death of me!

**Comment by andrewgarner** [comment]
:+1:

**Pull request merged**